api/observability/v1/clusterlogforwarder_types.go

```diff
@@ -0,0 +1,250 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ClusterLogForwarderSpec defines the desired state of ClusterLogForwarder
+type ClusterLogForwarderSpec struct {
+	// Specification of the Collector deployment to define
+	// resource limits and workload placement
+	//
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Collector Resources and Placement",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced"}
+	Collector *CollectorSpec `json:"collector,omitempty"`
+
+	// Inputs are named filters for log messages to be forwarded.
+	//
+	// There are three built-in inputs named `application`, `infrastructure` and
+	// `audit`. You don't need to define inputs here if those are sufficient for
+	// your needs. See `inputRefs` for more.
+	//
+	// +kubebuilder:validation:Optional
+	// +listType:=map
+	// +listMapKey:=name
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Log Inputs"
+	Inputs []InputSpec `json:"inputs,omitempty"`
+
+	// Outputs are named destinations for log messages.
+	//
+	// +kubebuilder:validation:Required
+	// +listType:=map
+	// +listMapKey:=name
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Receiver Outputs"
+	Outputs []OutputSpec `json:"outputs,omitempty"`
+
+	// Filters are applied to log records passing through a pipeline.
+	// There are different types of filter that can select and modify log records in different ways.
+	// See [FilterTypeSpec] for a list of filter types.
+	//
+	// +kubebuilder:validation:Optional
+	// +listType:=map
+	// +listMapKey:=name
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Forwarder Pipeline Filters"
```

Comment:
Similarly here: Why use "Forwarder"?

```suggestion
	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Log Pipeline Filters"
```